.github/workflows/release.yml

```
@@ -9,12 +9,15 @@ on:
 
 jobs:
   publish:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
       - name: Check out
         uses: actions/checkout@v4
-
-      - name: Publish to Hex.pm
-        uses: erlangpack/github-action@v3
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: '27'
+          rebar3-version: '3.24'
+      - name: Publish to hex.pm
         env:
           HEX_API_KEY: ${{ secrets.HEX_API_KEY }}
+        run: rebar3 hex publish -r hexpm --yes
```

I'd just do just in case
```suggestion
        run: rebar3 compile
        run: rebar3 hex publish -r hexpm --yes
```
As hex might not trigger `asn1` compilation from `dns_erlang` and then the package might be incomplete (happened for `dns_erlang` v3.0.0).